src/ralph/ui/forms/deployment.py

```diff
@@ -180,6 +187,30 @@ def _validate_preboot(preboot, row_number):
         )
 
 
+def _validate_service(service_name, row_number):
+    try:
```

Comment:
``if not ServiceCatalog.objects.filter(name=service_name).exists(): raise forms.ValidationError()`` will be better.